modules/mobile/app/models/mobile/v0/adapters/immunizations.rb

```diff
@@ -24,6 +24,19 @@ def parse(immunizations)
           end
         end
 
+        def parse_location(facility, lh_id)
```

Comment:
Could this be in its own class? There might be a time we fetch a location outside of immunizations.